clusters/app.ci/assets/view_monitoring_rbac.yaml

```diff
@@ -0,0 +1,37 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: monitoring-viewers
+  users:
+  - wking
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      openshift.io/description: This cluster role has read-only access to Prometheus monitoring
+    name: view-monitoring
```

Comment:
that name is confusing, please rename to `get-all-namespaces` or something like that. Same for the description

Reply:
Actually it seems this role already exists, `cluster-monitoring-view` so the binding should be enough

Reply:
`get-all-namespaces` is an implementation detail, right?  Once [rhbz#1888033][1] gets fixed, we will pivot the implementation to whatever the new approach is.  I'd like the name to reflect the goal, not the implementation.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1888033

Reply:
> Actually it seems this role already exists, cluster-monitoring-view...

Nice.  I found a link in openshift-docs to support that, and updated the PR with 345aea6357 -> 2e6b50522c.